x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -201,9 +201,10 @@ export function InfraHostsViewProvider({ getService }: FtrProviderContext) {
     },
 
     setAlertStatusFilter(alertStatus?: AlertStatus) {
-      const buttons = {
-        [ALERT_STATUS_ACTIVE]: 'hostsView-alert-status-filter-active-button',
-        [ALERT_STATUS_RECOVERED]: 'hostsView-alert-status-filter-recovered-button',
+      const buttons: Record<AlertStatus | 'all', string> = {
+        active: 'hostsView-alert-status-filter-active-button',
+        recovered: 'hostsView-alert-status-filter-recovered-button',
+        untracked: 'hostsView-alert-status-filter-untracked-button',
```

Comment:
Why not keeping the constants but add the `ALERT_STATUS_UNTRACKED`?

```suggestion
      const buttons: Record<AlertStatus | 'all', string> = {
        [ALERT_STATUS_ACTIVE]: 'hostsView-alert-status-filter-active-button',
        [ALERT_STATUS_RECOVERED]: 'hostsView-alert-status-filter-recovered-button',
        [ALERT_STATUS_UNTRACKED]: 'hostsView-alert-status-filter-untracked-button',
```